dWorldServer/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
-set(DWORLDSERVER_SOURCES "ObjectIDManager.cpp"
-		"PerformanceManager.cpp"
-		"WorldServer.cpp")
+set(DWORLDSERVER_SOURCES
+	"ObjectIDManager.cpp"
+	"PerformanceManager.cpp"
+)
+
+add_library(dWorldServer ${DWORLDSERVER_SOURCES})
+add_executable(WorldServer "WorldServer.cpp")
+
+target_link_libraries(dWorldServer ${COMMON_LIBRARIES})
```

Comment:
Really not certain if I am happy with this, if those things need to be used outside of WorldServer they should be taken out of the WorldServer directory.

Reply:
They should be but that's outside the scope of this pr.  We needed to get things compiling and this is the simplest change to do so.  Moving these files would also help us resolve the duplicate objectidmanagers we have.  I see no reason we need to entirely separate managers.

Reply:
We also need to do it like so so we can actually test these files in the future.  If left as is in main we could not test these files